style.css

```diff
@@ -21,19 +76,1060 @@
 
 ul {
   list-style: none;
+  margin-top: 0;
 }
 
-a {
+/*
+    ==============================================
+                ****** GLOBAL ******
+    ==============================================
+*/
+
+.body {
+  margin: 0;
+  font-family: var(--bs-font-sans-serif);
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+  color: #212529;
+  background-color: #fff;
+  -webkit-text-size-adjust: 100%;
+  -webkit-tap-highlight-color: transparent;
+}
+
+/* header */
+
+.bg-white {
+  background-color: #fff !important;
+}
+
+h3 {
+  display: block;
+  font-size: 1.17em;
+  margin-block-start: 1em;
+  margin-block-end: 1em;
+  margin-inline-start: 0;
+  margin-inline-end: 0;
+  font-weight: bold;
+}
+
+.p-0 {
+  padding: 0 !important;
+}
+
+.pe-4 {
+  padding-right: 1.5rem !important;
+}
+
+.m-0 {
+  margin: 0 !important;
+}
+
+.ms-2 {
+  margin-left: 0.5rem !important;
+}
+
+.me-3 {
+  margin-right: 1rem !important;
+}
+
+.ms-3 {
+  margin-left: 1rem !important;
+}
+
+.my-auto {
+  margin-top: auto !important;
+  margin-bottom: auto !important;
+}
+
+.container-fluid {
+  width: 100%;
+  padding-right: var(--bs-gutter-x, 0.75rem);
+  padding-left: var(--bs-gutter-x, 0.75rem);
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.container-xl {
+  width: 100%;
+  padding-right: var(--bs-gutter-x, 0.75rem);
+  padding-left: var(--bs-gutter-x, 0.75rem);
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.page-header {
+  position: relative;
+  z-index: 10;
+}
+
+.header {
+  display: block;
+}
+
+.header-bottom {
+  background-color: #f72210;
+  border-bottom: 2px solid #eee;
+}
+
+.header-bottom::before {
+  width: 66px !important;
+  position: absolute;
+  top: 92px;
+  right: 0;
+  content: "";
+  display: block;
+  background-color: #f72210;
+  border-left: 12px solid white;
+  border-top: 13px solid #f72210;
+}
+
+.header-bottom .mid-nav a {
+  text-decoration: none;
+  font-family: "Roboto Condensed", sans-serif;
+  color: #111;
+  font-size: 1.266rem;
+  font-weight: 700;
+}
+
+input {
+  -webkit-writing-mode: horizontal-tb !important;
+  text-rendering: auto;
+  letter-spacing: normal;
+  word-spacing: normal;
+  text-transform: none;
+  text-indent: 0;
+  text-shadow: none;
+  display: inline-block;
+  text-align: start;
+  appearance: auto;
+  cursor: text;
+  padding: 1px 2px;
+  border-width: 2px;
+  border-style: inset;
+  border-image: initial;
+  margin: 0;
+  font-size: inherit;
+  line-height: inherit;
+}
+
+.newsletter-signup .free-signup input {
+  width: 100%;
+  height: 32px;
+  line-height: 32px;
+}
+
+.header-bottom .mid-nav .mid-nav-search input {
+  width: 60px;
+  height: 54px;
+  outline: none;
+  border: none;
+}
+
+.bottom-nav a :hover {
+  color: #f72210;
+}
+
+.header-bottom .mid-nav :hover {
+  color: #f72210;
+}
```

Comment:
Keep your CSS rules DRY (Do not Repeat Yourself).
The rules above can be merged to become one. e.g.
```CSS
.bottom-nav a :hover,
.header-bottom .mid-nav :hover {
  color: #f72210;
}
```

Kindly fix this and check other for occurrences where you might be having code repetition.